op-scim-bridge/chart/op-scim-bridge-mp/Chart.yaml

```diff
@@ -15,7 +15,7 @@ icon: https://1password.com/img/logo-v1.svg
 sources:
   - https://app-updates.agilebits.com/product_history/SCIM
 # appVersion should match the value of publishedVersion in the schema.yaml
```

Comment:
```suggestion
# appVersion should match the value of publishedVersion in the schema.yaml and should equal the docker image version
```

Reply:
Or we should take out both of these doc strings and put them in the README

Reply:
addressed in 91d0beebf3b3222cce953cc7e8a11a01ae96ae33